### ISSUE TITLE
Strip tags from the page content before searching query string

### ISSUE THREAD

**Issue by maxlysenko** [getgrav/grav-plugin-simplesearch#21]
SimpleSearch doesn't find results because of html tags in page content.
For example we have a page with the following markdown content:
```md
## Grav **is** _awesome_!
```
Then if we try to search `grav is awesome`, there will be no results because `$cpage->content()` returns:
```html
<h2>Grav <strong>is</strong> <em>awesome</em>!</h2>
```
But if we strip tags from content, e.g. `strip_tags($cpage->content())` the query string `grav is awesome` will be found.

**Comment by rhukster** [comment]
Pull Request? :)

**Comment by rhukster** [comment]
Released in v1.6.1
